lib/gooddata/cli/hooks.rb

```diff
@@ -21,6 +22,9 @@
     username = creds[:username] if username.nil?
     password = creds[:password] if password.nil?
     token = creds[:auth_token] || creds[:token] if token.nil?
+    verify_ssl = (creds[:verify_ssl].nil? ? global[:verify_ssl] : creds[:verify_ssl]).to_s == 'true'
```

Comment:
The SSL-related logic is something we should have a test for.

Reply:
It's just a simple prioritized default value that the GLI library has no support for.

Reply:
I'm not sure we're talking about the same thing. I mean write unit tests to ensure that ssl is always verified unless the user says otherwise.

Reply:
Yes, we are talking about the same thing. I don't see the benefit of what you are proposing, especially since there is NO unit tests for this functionality at all.